src/tools/ceph_objectstore_tool.cc

```diff
@@ -2228,6 +2229,7 @@ void usage(po::options_description &desc)
     cerr << "ceph-objectstore-tool ... <object> dump" << std::endl;
     cerr << "ceph-objectstore-tool ... <object> set-size" << std::endl;
     cerr << "ceph-objectstore-tool ... <object> remove-clone-metadata <cloneid>" << std::endl;
+    cerr << "ceph-objectstore-tool --op apply-layout-settings (--pool <pool-name>|--pgid <pgid>) " << std::endl;
```

Comment:
@jdurgin A user could infer this usage because apply-layout-settings is listed as an --op in the auto generated usage and you have the message "apply-layout-settings requires either --pool or --pgid."   This line comes out under positional syntax, so I would remove this line.